scripts/webpack/webpack-resources.js

```diff
@@ -221,33 +297,56 @@ module.exports = {
         },
 
         plugins: [
-          // TODO: will investigate why this doesn't work on mac
-          // new WebpackNotifierPlugin(),
           ...(!process.env.TF_BUILD ? [new ForkTsCheckerWebpackPlugin()] : []),
-          // ...(process.env.TF_BUILD ? [] : [new webpack.ProgressPlugin()]),
+          ...(process.env.TF_BUILD || process.env.LAGE_PACKAGE_NAME ? [] : [new webpack.ProgressPlugin()]),
           ...(!process.env.TF_BUILD && process.env.cached ? [new HardSourceWebpackPlugin()] : []),
         ],
       },
       customConfig,
     );
 
-    if (!config.entry) {
-      // Handle packages which have a legacy demo app in the examples package
-      const packageName = path.basename(process.cwd());
-      const demoEntryInExamples = path.resolve(
-        __dirname,
-        '../../packages/react-examples/src',
-        packageName,
-        'demo/index.tsx',
-      );
-      if (fs.existsSync(demoEntryInExamples)) {
-        config.entry = demoEntryInExamples;
-      }
-    }
-
     config.entry = createEntryWithPolyfill(config.entry, config);
     return config;
   },
+
+  /**
+   * Create a serve config for a package with a legacy demo app in the examples package at
+   * `packages/react-examples/src/some-package/demo/index.tsx`.
+   * Note that this assumes a base directory (for serving and output) of `dist/demo`.
+   * @returns {WebpackServeConfig}
+   */
+  createLegacyDemoAppConfig() {
+    const packageName = path.basename(process.cwd());
+
+    const reactExamples = path.join(gitRoot, 'packages/react-examples');
+    const demoEntryInExamples = path.join(reactExamples, 'src', packageName, 'demo/index.tsx');
+
+    if (!fs.existsSync(demoEntryInExamples)) {
+      throw new Error(`${packageName} does not have a legacy demo app (expected location: ${demoEntryInExamples})`);
+    }
+
+    return module.exports.createServeConfig({
+      entry: {
+        'demo-app': demoEntryInExamples,
+      },
+
+      output: {
+        path: path.join(process.cwd(), 'dist/demo'),
```

Comment:
this is not quite right; http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/15414/merge/public-docsite-resources/demo/demo-app.js is 404

Reply:
I briefly got excited upon seeing the green builds...but yup the site is broken. I just deleted the existing version from blob storage so I can run a new PR build and fully see what it's doing.